tests/exchange/test_exchange_utils.cairo

```diff
@@ -0,0 +1,50 @@
+use starknet::{
+    ContractAddress, get_caller_address, get_contract_address, Felt252TryIntoContractAddress,
+    contract_address_const
+};
+use starknet::info::get_block_number;
+use snforge_std::{declare, start_prank, stop_prank, start_roll, ContractClassTrait};
+
+use satoru::data::keys;
+use satoru::data::data_store::{IDataStoreDispatcher, IDataStoreDispatcherTrait};
+use satoru::exchange::exchange_utils::validate_request_cancellation;
+use satoru::tests_lib::{setup, teardown};
+
+#[test]
+fn test_exchange_utils() {
```

Comment:
rename test to follow our naming conventions

ex: `given_division_by_0_when_roundup_magnitude_division_then_fails`